modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -830,6 +830,9 @@ export const console: ConsoleNS = {
                                     hint: "Cannot retrieve these user attributes by requesting " +
                                             "OIDC scopes. To retrieve, add the required attributes to a relevant scope."
                                 },
+                                selectedScopesComponentHint: "Request these scopes from your application to retrieve the user " +
+                                    "attributes you selected.",
+                                howToUseScopesHint: "<1>How to use Scopes</1>",
```

Comment:
get it reviewd by docs team

Reply:
Getting reviewed by @divyaamunugama

Reply:
@mpmadhavig 

Shall we update the text as follows?

`Request these scopes from your application to retrieve the selected user attributes. Learn how to <use scopes in your application>`

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3640/commits/36d8f442d1333ae449424157d70d879c508e5d43